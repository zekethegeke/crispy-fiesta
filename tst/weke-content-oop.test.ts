import dedent from 'ts-dedent';

test('all your object belong to us', () => {
    const content = dedent `
        === Purpose ===
    `;
    expect(content).toMatchSnapshot();
});