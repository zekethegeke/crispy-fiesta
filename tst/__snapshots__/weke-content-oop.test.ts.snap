// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`all your object belong to us 1`] = `"=== Purpose ==="`;
